scipy/fft/tests/test_basic.py

```diff
@@ -364,7 +368,7 @@ def test_dtypes_complex(self, dtype, xp):
 
         res_fft = fft.ifft(fft.fft(x))
         # Check both numerical results and exact dtype matches
-        rtol = {"complex64": 1.2e-4, "complex128": 1e-8}[dtype]
+        rtol = {"complex64": 1.2e-4, "complex128": 1e-7}[dtype]
```

Comment:
Could add `complex64` and `complex128` into the default tolerance determination.